### PR TITLE
[ADD] IoT: New doc about how to connect a scale using the IoT box.

### PR DESCRIPTION
**Pull request by Lau-Odoo**
Here is new documentation about connecting a [electronic] scale to the PoS app using an IoT box. 
The doc is good for 12.0 and after. Some new pieces of information could be added later to the latest version only.

### CODE REVIEW

File: content/applications/productivity/iot/devices/scale.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+.. image:: scale/driver-list.png
+   :align: center
+   :alt: View of the IoT box settings and driver list.
+
+Use a Scale in Point of Sale
+============================
+
+To use the scale in your *Point of Sale app*, go to the *Point of Sale* menu, under *Configuration*
+and open a *Point of Sale* in the list. Once opened, edit it and check the *IoT Box* feature.
```
Comment (jcs-odoo) [inline content/applications/productivity/iot/devices/scale.rst:35]:
```suggestion
To use the scale in your *Point of Sale* app, go to :menuselection:`Point of Sale --> Configuration
--> Point of Sale`, open the one you want to configure, then click on *Edit* and enable the *IoT
Box* feature.

```

File: content/applications/productivity/iot/devices/scale.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Now, choose the *IoT Box* in the dropdown menu and check the *Electronic Scale* option. Then, you
+hit save.
+
+.. image:: scale/electronic-scale-feature.png
+   :align: center
+   :alt: List of the external tools that can be used with PoS and the IoT box.
+
+The scale is now available in all your *PoS* sessions. Then, if a product has a price per weight
+set, clicking on it on the *PoS* screen will open the scale screen, where the cashier can weigh the
+product and add the correct price to the cart.
```
Comment (jcs-odoo) [inline content/applications/productivity/iot/devices/scale.rst:50]: nitpicking:
I know it's obvious that it's only the sessions of the configured PoS but you wxrite "all" pos sessions.

+

```suggestion
The scale is now available in all your *PoS* sessions. Then, if a product has a price per weight
set, clicking on it on the *PoS* screen opens the scale screen, where the cashier can weigh the
product and add the correct price to the cart.
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
really nice :)

a few suggestions then it's good to go

File: content/applications/productivity/iot/devices/scale.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+To connect the scale to the *IoT Box*, you just have to combine the two by cable.
+
+.. note::
+        In some cases, a serial to USB adapter may be needed.
+
+If your scale is a `Supported devices <https://www.odoo.com/page/iot-hardware>`__, there is no
+need to set up anything because it will be automatically detected as soon as it is connected.
+
+.. image:: scale/choix-iot.png
```
[the remark this answers is not in the document]
Comment (Lau-Odoo) [reply content/applications/productivity/iot/devices/scale.rst:20]: Pouahaha, sorry... 😆

**Review by jcs-odoo — APPROVED** [review]
lgtm 👍

**Review by AntoineVDV — APPROVED** [review]

**Pull request merged**